src/main/java/dev/sigstore/rekor/client/RekorVerifier.java

```diff
@@ -44,6 +43,7 @@ public static RekorVerifier newRekorVerifier(byte[] rekorPublicKey)
   private RekorVerifier(PublicKey rekorPublicKey, String verifierAlgorithm) {
     this.rekorPublicKey = rekorPublicKey;
     this.verifierAlgorithm = verifierAlgorithm;
+    this.calculatedLogId = Hashing.sha256().hashBytes(rekorPublicKey.getEncoded()).toString();
```

Comment:
just FYI that Asra was telling me this probably won't work long term with changes coming down the pipe. I think it's fine for now.

Reply:
makes sense. we can deal with that as it happens. It's not an essential check either.